### DISCUSSION TITLE
Avalonia Github Icon Page Not Responding

### DISCUSSION THREAD

**Discussion by FatemehRafiee1** [AvaloniaUI/Avalonia#16040] [Q&A]
Hi, I'm new to Avalonia. Yesterday I was using the https://avaloniaui.github.io/icons.html page to copy some icons that I need in my app and it was fine. However, now the url doesn't seem to work properly (showing 404 error). 
Just wanted to ask if there's another way that I can get the icons, cause I searched and didn't find anything useful.
Also I visited https://github.com/jmacato/fluentui-system-icons/blob/master/icons.md but it was apparently IOS and Android and didn't have anything like `<StreamGeometry x:Key="line_horizontal_3_regular">M2 4.5C2 4.22386 2.22386 4 2.5 4H17.5C17.7761 4 18 4.22386 18 4.5C18 4.77614 17.7761 5 17.5 5H2.5C2.22386 5 2 4.77614 2 4.5Z M2 9.5C2 9.22386 2.22386 9 2.5 9H17.5C17.7761 9 18 9.22386 18 9.5C18 9.77614 17.7761 10 17.5 10H2.5C2.22386 10 2 9.77614 2 9.5Z M2.5 14C2.22386 14 2 14.2239 2 14.5C2 14.7761 2.22386 15 2.5 15H17.5C17.7761 15 18 14.7761 18 14.5C18 14.2239 17.7761 14 17.5 14H2.5Z</StreamGeometry>` that I was after.

Thank you

![image](https://github.com/AvaloniaUI/Avalonia/assets/97700019/b2ae9c8d-b028-44d0-937f-b75f5b7ce32f)

**Comment by timunie** [comment]
I love to use MahApps.Metro.IconPacks.Browser to search icons and copy the path data from them. Make sure you check the licenses of each IconPack you use carefully

https://github.com/MahApps/IconPacks.Browser

**Reply by FatemehRafiee1** [reply]
Thank you, I'll definately check it out.

**Comment by thevortexcloud** [comment]
For the record, the Avalonia icons are just a slight fork of the Microsoft ones. Anyway, the full list from the Avalonia repo is here:
https://github.com/AvaloniaUI/fluentui-system-icons/blob/master/icons.md

And here is the Microsoft repo: https://github.com/microsoft/fluentui-system-icons

I think there is also some site that lets you view them too in an easy way, but I have not used it.

**Reply by FatemehRafiee1** [reply]
Thanks, I checked them out but still have no idea how I can get their `<StreamGeometry>` format.

**Reply by thevortexcloud** [reply]
Pretty sure that's just the vector format. If you look at the raw SVG data in a text editor you can get the same information.

EG https://github.com/AvaloniaUI/fluentui-system-icons/blob/d66faf3ad7392a52f74e90296f0872565f75561b/assets/Arrow%20Step%20In/SVG/ic_fluent_arrow_step_in_16_filled.svg#L2

Has this:

```xml
 <path d="M7.5 8.29289L7.5 1.5C7.5 1.22386 7.72386 1 8 1C8.27614 1 8.5 1.22386 8.5 1.5L8.5 8.29289L10.6464 6.14645C10.8417 5.95118 11.1583 5.95118 11.3536 6.14645C11.5488 6.34171 11.5488 6.65829 11.3536 6.85355L8.35355 9.85355C8.25979 9.94732 8.13261 10 8 10C7.86739 10 7.74021 9.94732 7.64645 9.85355L4.64645 6.85355C4.45118 6.65829 4.45118 6.34171 4.64645 6.14645C4.84171 5.95118 5.15829 5.95118 5.35355 6.14645L7.5 8.29289ZM8 15C9.10457 15 10 14.1046 10 13C10 11.8954 9.10457 11 8 11C6.89543 11 6 11.8954 6 13C6 14.1046 6.89543 15 8 15Z" fill="#212121"/>
```

I could be wrong though, vector formats are not something I have even looked too much into.

**Comment by kyuranger** [comment]
I prefer use this icon library designed by Google Font: https://fonts.google.com/icons?icon.query=menu&icon.size=24&icon.color=%235f6368
